src/server/internalServer.cpp

```diff
@@ -442,6 +441,19 @@ std::unique_ptr<Response> InternalServer::handle_suggest(const RequestContext& r
     return Response::build_404(*this, request, bookName, "");
   }
 
+  auto start = 0;
+  try {
+    start = request.get_argument<unsigned int>("start");
+  } catch (const std::exception&) {}
+
+  unsigned int suggestionLength = 10;
+  try {
+    suggestionLength = request.get_argument<unsigned int>("suggestionLength");
+  } catch (const std::exception&) {}
+  if (suggestionLength == 0) suggestionLength = 10;
```

Comment:
Just a matter of style but I prefer :
```cpp
if (suggestionLength == 0) {
    suggestionLength = 10;
}
```

As for `pageLength` in `handle_search`

(Yes, I must write a coding style guide)

Reply:
Aah right, you mentioned this before :sweat_smile: 

Casually tagging openzim/libzim#513, we should really document our coding style.